flow-bridge/src/main/java/org/reactivestreams/ReactiveFlowBridge.java

```diff
@@ -0,0 +1,357 @@
+/************************************************************************
+ * Licensed under Public Domain (CC0)                                    *
+ *                                                                       *
+ * To the extent possible under law, the person who associated CC0 with  *
+ * this code has waived all copyright and related or neighboring         *
+ * rights to this code.                                                  *
+ *                                                                       *
+ * You should have received a copy of the CC0 legalcode along with this  *
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.*
+ ************************************************************************/
+
+package org.reactivestreams;
+
+import java.util.concurrent.Flow;
+
+/**
+ * Bridge between Reactive Streams API and the Java 9 {@link java.util.concurrent.Flow} API.
+ */
+public final class ReactiveFlowBridge {
```

Comment:
Same as commented previously, `ReactiveStreamsFlowBridge`. As this is directly visible user API

Reply:
Sure.